python/lsst/ts/standardscripts/auxtel/calibrations/power_on_atcalsys.py

```diff
@@ -0,0 +1,312 @@
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+__all__ = ["PowerOnATCalSys"]
+
+import asyncio
+import time as time
+
+import yaml
+
+from lsst.ts.idl.enums import ATWhiteLight
+from lsst.ts.observatory.control.remote_group import Usages
+
+
+from lsst.ts import salobj
+
+track_lamp_warmup = 60
+
+
+class PowerOnATCalSys(salobj.BaseScript):
+    """Powers on the ATCalSys dome flat illuminator
+    (ATWhiteLight and ATMonochromator)
+    required to perform image calibrations over white light.
+
+    Parameters
+    ----------
+    index : `int`
+        Index of Script SAL component.
+
+    Notes
+    -----
+
+    """
+
+    def __init__(self, index, add_remotes: bool = True):
+        super().__init__(
+            index=index,
+            descr="Power On AT Calibration System ",
+        )
+
+        self.white_light_source = None
+        self.monochromator = None
+
+        # White lamp config
+        self.timeout_lamp_warm_up = 60 * 15
+        self.cmd_timeout = 30
+
+        # Chiller config
+        self.timeout_chiller_cool_down = 60 * 10
+        self.chiller_temp_tolerance_relative = 0.1
+
+        if self.white_light_source is None:
+            self.white_light_source = salobj.Remote(
+                domain=self.domain, name="ATWhiteLight", 
+                intended_usage= None if add_remotes else Usages.DryTest, 
+                log =self.log
+            )
+
+        if self.monochromator is None:
+            self.monochromator = salobj.Remote(
+                domain=self.domain, name="ATMonochromator", 
+                intended_usage= None if add_remotes else Usages.DryTest, 
+                log=self.log
+            )
+
+        asyncio.gather(
```

Comment:
You need to `await` this otherwise nothing happens.. Once you move this to the `configure` method, you will be able to do:

```
await asyncio.gather(...)
```